pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -55,12 +60,16 @@ public JwtAuthenticator(final String secret) {
     public void validate(TokenCredentials credentials) {
         CommonHelper.assertNotBlank("secret", secret);
 
-        final String token = credentials.getToken();
+        String token = credentials.getToken();
         boolean verified = false;
         SignedJWT signedJWT = null;
 
         try {
             // Parse the token
+            if (Base64.isBase64(token)) {
```

Comment:
I don't thinking creating new methods is really an option if we want to use generic JWT tokens in a normal pac4j flow. Forget this suggestion.